easybuild/easyblocks/p/picard.py

```diff
@@ -72,13 +72,106 @@ def install_step(self):
 
     def sanity_check_step(self):
         """Custom sanity check for picard"""
-        jar_files = ['picard']
-        if LooseVersion(self.version) < LooseVersion('1.115'):
-            jar_files.append('sam')
+        """All versions prior to 1.124 have these jar files"""
+        if LooseVersion(self.version) < LooseVersion('1.124'):
+            jar_files = [
+                'picard-%s' % self.version,
+                
```

Comment:
I'm willing to accept spreading this vertically, but let's not insert blank lines, this is too confusing...